config/maps.txt

```diff
@@ -44,6 +44,10 @@ map kilostation
 	votable
 endmap
 
+map ministation
+	votable
```

Comment:
No maxplayers?

Reply:
Set to 25 for now but I feel 20 would be better